distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -277,11 +284,13 @@ atom_with_hole factory = @Tail_Call Meta_Helpers.atom_with_hole_builtin factory
 ## PRIVATE
    ADVANCED
 
-   Returns a meta-representation of a given runtime entity.
+   Returns a meta-representation of a given runtime entity. If the entity
+   isn't public (like a private constructor), then an `Private_Access` error
```

Comment:
- to satisfy the _"viewing from a distance"_ mantra stated in this PR
- we should apply this consistently to `type` definitions as well
- obtaining non-public types should be forbidden
- and yield `Private_Access` error
- I am not sure I want to make such change in this PR however...